bloom-and-vine/src/Pages/Contact.css

```diff
@@ -0,0 +1,204 @@
+@import url('https://fonts.googleapis.com/css?family=Italiana');
+@import url('https://fonts.googleapis.com/css?family=Cormorant Garamond');
+
+.Contact {
+  font-family: 'Italiana';
+  flex-wrap: nowrap;
+  margin: 0 auto;
+  width: auto;
+  height: 100vh;
+
+  /* min-height: 100%; */
+  /* display: table; */
+}
+
+.Rightside-Screen {
+  /* background: lightpink; */
+  /* float: right; */
+  width: auto;
+  /* height: 1542px;  */
+  position: relative;
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  
+}
+
+.Leftside-Screen {
+  float: left;
+  position: relative;
+  background: #FAF1EF;
+  width: auto;
+  height: 1542px;
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.form-wrapper {
+  text-align: left;
+  margin-left: 250px;
+  line-height: 80px;
+  font-weight: 700;
+  font-size: 24px;
+  padding-top: 150px;
+}
+
+input[type=text], select, textarea {
+  font-family: 'Italiana';
+  display: block;
+  width: 622px;
+  height: 55px;
+  border: 2px solid #000000;
+  border-radius: 10px;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+input[type=date] {
+  font-family: 'Italiana';
+  display: block;
+  width: 622px;
+  height: 55px;
+  border: 2px solid #000000;
+  border-radius: 10px;
+  font-size: 24px;
+  font-weight: bold;
+
+}
+
+::placeholder {
+  text-decoration-color: grey;
+}
+
+textarea {
+  width: 622px;
+  height: 300px;
+}
+
+/* Style the submit button with a specific background color etc */
+input[type=submit] {
```

Comment:
Actually, all of the inputs are too close to the edge of the textbox. add some space on the left side to add padding